setup.py

```diff
@@ -21,7 +21,7 @@
         'navmazing',
         'python-box',
         'pytest',
-        'selenium',
+        'selenium==4.1.0',
```

Comment:
Currently, selenium is being installed in the latest version, but you are trying to freeze this to a specific version!  This comes an additional overhead of upgrading it every time to the suitable version!

Why not stick to the existing approach? Does `selenium` installs a different version than selenium 4 ?

Reply:
This is the latest version where I have tested it with rehat image. I think this will have not any value if I guess https://github.com/RedHatQE/widgetastic.patternfly4/pull/128 PR get merged

Reply:
> pip list | grep selenium
selenium                4.1.0
selenium-smart-locator  0.2.0
> pip list | grep pattern
widgetastic.patternfly  1.3.4
widgetastic.patternfly4 0.23.2.dev2+gcb3bf05